templates/core/terraform/firewall/locals.tf

```diff
@@ -0,0 +1,9 @@
+locals {
```

Comment:
Can we pull these out to variables with defaults? I expect customers will need to add to these - especially for Nexus. 

Don't really want to have to edit the locals file?

Can make a separate issue on the backlog - not urgent.

Reply:
@marrobi pushed an update for this. This isn't "wired" end to end with env variables but it's almost there for the TF part.